.travis.yml

```diff
@@ -14,8 +14,8 @@ script:
       export DOCKER_HUB_OPERATOR_TAG=23.7.0-dev
       export DOCKER_HUB_OPERATOR_TEST_TAG=23.7.0-dev
     else
-      export DOCKER_HUB_OPERATOR_TAG=`git rev-parse --short HEAD`
-      export DOCKER_HUB_OPERATOR_TEST_TAG=`git rev-parse --short HEAD`
+      export DOCKER_HUB_OPERATOR_TAG="git rev-parse --short HEAD"
+      export DOCKER_HUB_OPERATOR_TEST_TAG="git rev-parse --short HEAD"
```

Comment:
uh.. these are "shell-executable backticks" that actually "need to run"

So, we'd need to replace them with `$(...)` -->

```
      export DOCKER_HUB_OPERATOR_TAG=$(git rev-parse --short HEAD)
      export DOCKER_HUB_OPERATOR_TEST_TAG=$(git rev-parse --short HEAD)
```

Reply:
updated